shared/common-adapters/copy-text.js

```diff
@@ -1,27 +1,13 @@
 // @flow
 import * as React from 'react'
 import * as ConfigGen from '../actions/config-gen'
-import {Box2} from './box'
-import Icon from './icon'
-import Button from './button'
-import ButtonBar from './button-bar'
-import Text from './text'
-import Toast from './toast'
+import {Box2, Icon, Button, ButtonBar, Text, Toast} from '.'
```

Comment:
I think we want to avoid this since it's a circular dependency

Reply:
Good catch! I was getting an error when I tried to add the `HOCTimers` component to this line. Guess circular dependencies ruin my attempts to reduce lines 🙃